extjsdk/src/main/java/io/vantiq/extjsdk/ExtensionWebSocketListener.java

```diff
@@ -62,7 +61,7 @@
      * this listener from {@link ExtensionWebSocketClient} or B) the client has already successfully authed. Set by
      * {@link #setOverrideHandler}
      */
-    private Handler<Map> overrideHandler = null;
+    private Handler<Map<String,Object>> overrideHandler = null;
```

Comment:
This handler is for if the user decides "screw your listener logic, I want to figure it out on my own and do weird custom stuff", so it handles every message before they are even turned into a Response or ExtSvcMsg. Now that I'm thinking about it again it *might* be better and cleaner to just remove it, because I doubt many people would even want to use this option

Reply:
yeah, if they want that, they can skip the SDK.  There's certainly no requirement to use it.